notebooks/10-introduction-raster.md

```diff
@@ -36,325 +35,234 @@ __Raster data__ is stored as a grid of values which are rendered on a map as pix
+A raster is an __image__ in local pixel coordinates until we specify what part of the earth the image covers. This is done through the usage of raster file metadata. It depends on the file type how these are stored on disk. When reading in a file, the metadata need to be interpreted as well in order to know the __spatial information__. The translation from the data and metadata to a Python Nd-array with geographical context is done by the rasterio/xarray packages.
 
-</div>
+As an quick introduction (we will learn the specific syntax later).
 
-+++
+> _A plot of the Red band only, using a Red color scale_
 
-Important information we get from the `gdalinfo` command are 
+```{code-cell} ipython3
+ax = herstappe.sel(band=1).plot.imshow(cmap="Reds", figsize=(12, 5))  #, robust=True)
+ax.axes.set_aspect('equal')
+```
 
-* Coordinate reference system (CRS), see also [02-coordinate-reference-systems notebook](./02-coordinate-reference-systems.ipynb).
-* Number of bands in the data set: A data set can contain one (single) or more (multi) layers (bands).
-* Spatial resolution: This resolution represents the area on the ground that each pixel covers. The units for your data are determined by the CRS above (e.g. meters).
-* Spatial extent (bounding box) of the data: The geographic area that the raster data covers.
+> _plot the intersection of the data at x coordinate closest to 600000 for each band:_
 
-<img src="../img/raster-spatial-extent-coordinates.png" alt="spatial-extnet" style="width:500px;"/>
+```{code-cell} ipython3
+herstappe.sel(x=600_000, method='nearest').plot.line(col='band')
+```
 
-<small>_Image Source: National Ecological Observatory Network (NEON)._</small>
+## What's a xarray.DataArray?
 
 +++
 
-Let's see how Rasterio interprets this information:
-
-```{code-cell} ipython3
-src = rasterio.open(file_herstappe)
-```
+Let's have a look at the data again:
 
 ```{code-cell} ipython3
-src.meta
+herstappe
 ```
 
-The `meta` attribute contains some of the essential spatial information, whereas the `profile` attribute contains the spatial metadata combined with the metadata about the GeoTiff storage:
+The output of xarray is a bit different to what we've previous seen. Let's go through the different elements:
 
-```{code-cell} ipython3
-src.profile
-```
+- It is a `xarray.DataArray`, one of the main data types provided by xarray
+- It has 3 __dimensions__:
+    - `band`: 3 bands (in this case RGB)
+    - `y`: the y coordinates of the data set
+    - `x`: the x coordinates of the data set
+- Each of these dimensions are defined by a __coordinate__ (1D) array
+- It contains 304407 (227*447*3) data values (stored as float32)
+- Other metadata, such as the spatial information) provided by the `tiff` are stored in the __`Attributes`__
 
-Some of these are also stored in a separate attribute:
+Looking to the data itself (click on the icons on the right), we can see these are Numpy arrays.
```

Comment:
This is a bit duplicated with the content of "## Xarray adds (spatial) context to NumPy arrays"?

Reply:
Indeed, I've removed it from this section and rewritten the introduction of the next section.